projects/towify-table/src/lib/type/towify.table.type.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @Author  : xiongxianti
+ * @Date    : 2021/6/16
+ * */
+
+export type TableFieldValueType = 'id' | 'text' | 'yesOrNo' | 'number' | 'encryptedText' | 'date' | 'image' | 'audio' | 'video' | 'table'
+
+export type TableScrollDirectionType = 'up' | 'down' | 'left' | 'right' | 'unknown';
+
+export type TableColumnInfoType = {
+  name: string;
+  id: string;
+  type: TableFieldValueType;
+  iconName: string;
+  width?: number;
```

Comment:
export type TableColumnInfoType = {
    [key in string]: any
    key: string
}